### PR TITLE
master qa 72287

### PR DESCRIPTION
**Pull request by kyle-miho**
https://issues.liferay.com/browse/LRQA-72287

### CODE REVIEW

**Comment by kyle-miho** [comment]
This testcase covers a scenario on our Core Infrastructure test map to ensure customers could upgrade their service builder modules from using Spring's dependency Injector to OSGi DS. I created a test module to perform this since currently based off a CTRL + F we have no Liferay modules using Spring DI still (and this test could break that module)

As reported here, https://issues.liferay.com/browse/LPS-148859 this interaction does not currently work, I checked on CI here https://github.com/kyle-miho/liferay-portal/pull/479 Im sending this PR here to review to check over the actual test module to make sure it is created correctly

File: build-test.xml [hunk 1]
```diff
@@ -14370,6 +14374,14 @@ url=${short.hostname}:8080</echo>
 		/>
 	</target>
 
+	<target name="update-dependency-injector">
+		<replace
+			file="${project.dir}/${gradle.file.path}/service.xml"
+			token="dependency-injector=&quot;spring&quot;"
```
Comment (tinatian) [inline build-test.xml:14380]: @kyle-miho When you use spring as dependecy injector, you won't find "dependency-injector=spring" in your service.xml, as we added this property "dependency-injector" when we introduced ds dependency.

**Comment by tinatian** [comment]
@kyle-miho 
For how to change a spring dependency module to use ds, please read the description in this ticket https://issues.liferay.com/browse/LPS-89637. Also you can find an example if you take a look at the sub tasks.

@vicnate5 I am wondering why we want to do this test, as generally you can not finish the transfer to ds automatically based on the steps listed in this ticket (https://issues.liferay.com/browse/LPS-89637).


Thanks.
Tina.

**Comment by vicnate5** [comment]
@tinatian None of those manual steps couldn't be handled by a script since we are adding our own module to test it.

**Comment by tinatian** [comment]
@kyle-miho Please see my comment and resend. I am going to close this pull.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**